src/static/settings.html

```diff
@@ -0,0 +1,70 @@
+<html lang="en">
+
+<head>
+  <meta charset="utf8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="settings.css" type="text/css">
+  <link rel="stylesheet" href="content.css" type="text/css">
+  <link rel="stylesheet" href="hint.css" type="text/css">
+  <link rel="shortcut icon" href="logo/Tridactyl_64px.png" type="img/png">
+  <title>Tridactyl Settings</title>
+</head>
+
+<body class="container">
+  <h1 class="title">Tridactyl Settings</h1>
+  <nav>
+    <ul id="tabbar" class="nav nav-tabs" role="tablist">
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#current-settings">Current Settings</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#rc-text">.tridactylrc</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#about">About</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#license">License</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#changelog">Changelog</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" role="tab" href="#readme">Readme</a>
+      </li>
+    </ul>
+  </nav>
+  <div id="tab-container" class="tab-content">
+    <div class="tab-panel" role="tabpanel" id="current-settings"></div>
+    <div class="tab-panel" role="tabpanel" id="rc-text">
+      <p>
+        If the button to load from filesystem is not working, check the console
+        to see the error you are getting. Yes, we don't have an error reporting
+        mechanism right now.
+      </p>
+      <button id="btn-save-rc">Save to browser</button>
+      <button id="btn-load-rc">Load from browser</button>
+      <button id="btn-load-fs-rc">Load from filesystem</button>
```

Comment:
"Read config file from $XDG_CONFIG_HOME/tridactyl/tridactylrc"?

I think it also needs much more explanation saying that it requires a native messenger. Perhaps we can check to see if we can talk to the messenger on page load and grey out the button if we cannot, with some red text saying "Native messenger not found", etc?

I was 100% convinced your "Load from file system" was going to open a file picker dialogue.